src/components/pages/TeacherDashboard/TeacherViewSection/SectionRumbleList/SectionRumbleCard.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { useResetRecoilState, useSetRecoilState } from 'recoil';
+import { Rumbles, Sections } from '../../../../../api';
+import { useRumbleStatus } from '../../../../../hooks';
+import { current } from '../../../../../state';
+import { Button, Table } from '../../../../common';
+
+const SectionRumbleCard = ({
+  section,
+  rumble,
+}: ISectionRumbleCardProps): React.ReactElement => {
+  const { push } = useHistory();
+  const setCurrentSection = useSetRecoilState(current.section);
+  const setCurrentRumble = useSetRecoilState(current.rumble);
+  const clearCurrentStudent = useResetRecoilState(current.student);
+
+  const openRumble = () => {
+    setCurrentSection(section);
+    setCurrentRumble(rumble);
+    clearCurrentStudent();
+    push('/dashboard/teacher/rumble', { rumble, section });
+  };
+
+  //Setting date and day of the week
+  const newDate = new Date(rumble.end_time);
```

Comment:
Don't declare stateless variables in the scope of a component if possible, use Luxon